### PR TITLE
Update MM concept model background, project overview, literature review.

### PR DESCRIPTION
**Pull request by SantoniS**

### CODE REVIEW

File: docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst [hunk 1]
```diff
@@ -36,34 +36,70 @@ Vivarium - CSU - Simulating Multiple Myeloma registries
 +------------------------------------+
 | List of abbreviations              |
 +=======+============================+
-|       |                            |
+|   MM  | Multiple myeloma           |
++-------+----------------------------+
+|   RR  | Relapsed/ Refractory MM    |
```
Comment (yongqx) [inline docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst:41]: A more common abbreviation of relapsed/refractory multiple myeloma is "RRMM".

File: docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst [hunk 2]
```diff
@@ -36,34 +36,70 @@ Vivarium - CSU - Simulating Multiple Myeloma registries
...
+* Population with renal impairment 
 
+In the absence of data on treatment effects for sub-groups we will need to make assumptions with client guidance. A strength of this project is that we will be able to iteratively update such assumptions as we get real world data from registries and re-run the simulation. 
 
 .. _1.1:
 
 1.1 Project overview
 --------------------
-
+This project intends to model the impact of a new treatment option for MM, Isatuximab, from 2021 to 2026 among the general population, key subpopulations listed above, and a registry population within the United States. The model will make use of the current multiple myeloma treatment guidelines in the United States, which are dependent on cancer stage, age, comorbidities, eligibility for stem cell transplantation, and risk category.
```
Comment (yongqx) [inline docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst:67]: Isa. is a new type of treatment for RRMM instead of MM.

**Review by yongqx — APPROVED** [review]

File: docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst (hunk 2 above)
Comment (SantoniS) [reply docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst]: Thank you for the correction! I have made this update.

File: docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst (hunk 1 above)
Comment (SantoniS) [reply docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst]: I've updated the acronym to 'RRMM', thanks!

**Review by aflaxman — APPROVED** [review]

File: docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst [hunk 3]
```diff
@@ -36,34 +36,70 @@ Vivarium - CSU - Simulating Multiple Myeloma registries
...
++-------+----------------------------+
+|   isa.| Isatuximab                 |
 +-------+----------------------------+
 
+
 .. _1.0:
 
 1.0 Background
 ++++++++++++++
+For this project, we are interested in the expected impact of Isatuximab on the overall Unites States population, among a registry population, and in key-subpopulations listed below, in comparison to a business as usual treatment scenario over 5 years.
```
Comment (alibow) [inline docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst:51]: I wonder if we should make a note about the sub-aim for this project that Bill discussed of the "power calculation" type approach of how many patients would need to be in a registry population in order to observe a significant effect of a given magnitude, and how many patients per year out of all patients per year would need to be enrolled over X years in order to reach that target.

**Review by alibow — APPROVED** [review]
Looks good to me! :)

File: docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst (hunk 3 above)
Comment (SantoniS) [reply docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst:51]: That's a great note, thanks! I have added that in the 'Modeling aims and objectives' section.

**Pull request merged**